p2panda-rs/benches/graph.rs

```diff
@@ -0,0 +1,102 @@
+// Benchmarking tests adapted from [https://github.com/declanvk/incremental-topo/tree/main/benches](https://github.com/declanvk/incremental-topo/tree/main/benches).
```

Comment:
Do we need license headers for these? I don't know! But maybe we should add them, better to have them than not :-D